### DISCUSSION TITLE
Async rust support

### DISCUSSION THREAD

**Discussion by bestgopher** [Tencent/tquic#22] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by iyangsj** [comment]
@bestgopher The support for Async APIs will be available in upcoming versions.

**Reply by bestgopher** [reply]
how to join this?

**Reply by iyangsj** [reply]
Welcome to our TQUIC developer group for in-depth discussions.

**Reply by bocui23** [reply]
What's the plan of the upcoming async API release, any hints about the timeline?

**Reply by iyangsj** [reply]
We plan to support async API in March 2024.

**Reply by bocui23** [reply]
Good, thanks for the info.
